crates/transaction-emitter-lib/src/emitter/account_minter.rs

```diff
@@ -130,11 +130,10 @@ impl<'t> AccountMinter<'t> {
                     panic!("Aborting");
                 }
             } else {
-                let max_allowed = 2 * req
-                    .expected_max_txns
-                    .checked_mul(req.expected_gas_per_txn * req.gas_price)
+                let max_allowed = (2 * req.expected_max_txns as u128)
```

Comment:
this is a correct fix, thanks.

I misunderstood, thought that issue was for computing coins_for_source, and so that balance < coins_for_source would be the check in question, not this one.